sim/visuals/microbit.ts

```diff
@@ -72,6 +72,13 @@ namespace pxsim.visuals {
         pointer-events: none;
         }
 
+        .sim-text-small {
+            font-family:"Lucida Console", Monaco, monospace;
+            font-size:20px;
+            fill:#fff;
+            pointer-events: none;
+        }
+
```

Comment:
```suggestion
        .sim-text-small,
```

nit: seems like you can combine these two?

Reply:
Joey that is genius